README.md

```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
+BDB adds parts. Lots of parts. Nearly 500 parts, including launchers, spacecraft, and probes, all in a ReStockalike style. The focus lies primarily on the US space program up through the Apollo program, though some modern parts exist. They are not perfect replicas, but rather take inspiration directly from existing designs. Care has been taken to research and include pieces for projects that were proposed but never happened, to increase the possibilities available to the player. Parts are made in a lego frame of mind, and screenshots of your unique franken creations are always appreciated! The mod is not recommended for new players, as the mod adds a good deal of clutter to the VAB lists. The mod also adds new standard sizes (such a 0.9375m, 1.5m and 1.875m diameter) of parts in order to more accurately scale to KSP.
 
-Now, 400 parts sounds like it would kill your PC. While the mod certainly is large, the memory overhead is likely lower than you would expect. Care has been taken to ensure the mod is efficient, with methods such as enforcing a strict texel density (a technique I have only used since mid-2017, so the older assets might not be as efficient) and texture atlasing to cram several parts onto each texture sheet.
+Now, 500 parts sounds like it would kill your PC. While the mod certainly is large, the memory overhead is likely lower than you would expect. Care has been taken to ensure the mod is efficient, with methods such as enforcing a strict texel density (a technique I have only used since mid-2017, so the older assets might not be as efficient) and texture atlasing to cram several parts onto each texture sheet.
 
-For those familiar with KSP mods, BDB can be considered a stockalike equivalent of FASA, and a US counterpart to Tantares' soviet rockets. 
+For those familiar with KSP mods, BDB can be considered a ReStockalike equivalent of FASA, and a US counterpart to Tantares' soviet rockets. 
 
-Currently, the mod includes (but is not limited to) parts to make:
+Currently, the mod includes (but is not limited to):
 - Mercury spacecraft (Courtesy of Bloeting Aerospace Corp)
 - Gemini spacecraft (Courtesy of Trails Manufacturing Co.)
+- Gemini Ferry
+- Big Gemini 69' (nice)
 - Apollo Block I and II spacecraft, as featured in REAL LIFE!
 - Apollo Block III, III+, IV, and V spacecraft, from Eyes Turned Skywards
 - AARDV "Aardvark" autonomous resupply vehicle, also from Eyes
-- Modular 1.875m station parts for Gemini-era space stations
+- Modular 1.875m and 1.5m station parts for Gemini-era space stations
 - Skylab station parts
 - Spacelab (from Eyes) station parts
+- Parts for the Apollo-Venus mission
```

Comment:
That kind of sounds off, and the technical name was "Manned Venus Flyby", though you can also note it is part of the AAP.